assets/css/style.css

```diff
@@ -5,11 +5,11 @@
     margin-right: 3em;
 }
 
-/* .sidebar-nav li > a[href^="/"]:not([href*="?id="]):only-child,
+.sidebar-nav li > a[href^="/"]:not([href*="?id="]):only-child,
```

Comment:
for now this styling is a basic fix (spacing, removing chevron icon etc) but changing the chevron icon on arabic translation would be great , however i suspect this can be only done through JavaScript